### PR TITLE
New options for type of role

### PR DESCRIPTION
**Pull request by goatlady**
plus improved wording for team configuration

### CODE REVIEW

**Comment by snoopdougiedougie** [comment]
I'd like to see "Programmer-Writer" or something like that. It's for people like me who document SDKs, including writing simple code examples in various programming languages.

**Comment by goatlady** [comment]
> I'd like to see "Programmer-Writer" or something like that. It's for people like me who document SDKs, including writing simple code examples in various programming languages.

@snoopdougiedougie hmmm, interesting! I would have thought that code examples and the like are part of the role of writer/content creator when we're talking about documentation - dealing with technical content being one of the things that distinguishes technical writing from other forms of writing.  I'd like to hear other input on this. 🤔

**Comment by bob-watson** [comment]
I 2nd @snoopdougiedougie's comment and would like to see such an option.

I think the challenge/confusion WRT the `programmer-writer` term is that I've only seen it in large writing teams where there's been specialization into roles, such as: 

* Editor
* UX writer
* Technical writer (Admin docs and procedural content)
* Programmer-writer (docs and code-samples for dev audience)
* Doc ops (managing the production pipeline and maybe the servers as well)

While a tech writing job in a small shop might have a "technical writer" doing all of the aforementioned roles.

I think it's important to note that these titles and sub-categories have been used to differentiate salary ranges, so while the actual duties might vary, the relationship between title and salary might be more valuable to readers, than how precisely they describe actual duties. I don't know where things are at the moment, but a "programmer-writer" would generally be placed in higher salary band than a "content creator" from what I've seen in the past. I've definitely seen a salary advantage to job titles that sounded more "technical."

Maybe use this section to describe their tasks (i.e. what they do) and add an entry for "job title" (what they were called) to help find the correlations between titles and pay as well as tasks and pay?

**Comment by goatlady** [comment]
@bob-watson thank you for your input! We do already have a field for job title. This question about type of role was included because of the disparity between job title and what people actually do - in 2023, 32% of all respondents reported that their official job title was "technical writer" yet as you point out, in some situations that covers a wide range of different tasks. 

From the comments here and those on [the WTD Slack thread](https://writethedocs.slack.com/archives/C6D77HJ4F/p1722513572495449) I'm convinced: type of role needs to be far more granular and the current grouping of roles - writer, editor, content creator, producer together - is too broad.  I'll rework the question and put it up for more feedback.

**Comment by goatlady** [comment]
@snoopdougiedougie @bob-watson I've restructured this question and split it into two parts - one for primary role and one for additional roles, and I've expanded the options to include the writer roles that were previously lumped together. Would love your feedback (the diff is hard to follow - [check the complete file here](https://github.com/writethedocs/salary-survey/blob/adb4ad2ccb2389ca9a97f6efe0adf6dfeee10c35/a-employment-parameters.md)).
